monitoring/ingestion/ingestion-processor-alert.yaml

```
@@ -0,0 +1,31 @@
+x-inputs:
+  - name: namespace
+    type: constant
+  - name: job_ingestion_processor
+    type: constant
+  - name: replicas
+    type: constant
+
+groups:
+- name: IngestionProcessor
+  rules:
+
+  - alert: IngestionProcessorDown
+    Expr: sum(up{namespace="${namespace}", job="${job_ingestion_processor}"}) == 0
+    For:   "30s"
+    Labels:
+      severity: critical
+    Annotations:
+      zenko_service: backbeat-ingestion-processor
+      description: "All Ingestion Processor pod(s) has been down for 30 seconds"
```

What do you think of not including the time in the description? Since adjusting the "for" value would probably be a common thing clients modify.
```suggestion
      description: "All Ingestion Processor pod(s) are down"
```

---

I think it is fine and useful for the use to know how long the pods has been down for. 
The "for" value is part of the prometheus alert rule and hardcoded (at least for now).

---

ok, I wonder if there is a substitution allowed like `${for}` or something.